### ISSUE TITLE
A bunch of syntaxes not working

### ISSUE THREAD

**Issue by melinstagibson** [TheLimeGlass/Skungee#21]
```
command /proxytest:
        permission: *
        trigger:
                send "%bungee servers%"                 #works
                send bungee "%player%" to "lobby" #works
                send "%bungee players%"                 #works
                if bungee "%player%" is online:         #doesnt work
                        send "hi12" to bungee "%player%"
                send "hi" to all bungee players          #doesnt work
                send "test" to bungee "%player%"    #works

```
pretty much using the default config, paperspigot 1.13 and 1.12 (tried both), skript 2.3-alpha4 and skungee 1.0.7d

**Comment by melinstagibson** [comment]
the if statement returns false btw, not none

**Comment by Efnilite** [comment]
Skript 2.3 is still experimental and I believe that Skungee is only tested and made for 1.12, so you shouldn't be using Skript 2.3 if you're using Skungee. Use dev37c or dev36 since they probably will work. But @TheLimeGlass should know if I'm completely wrong.

Proof is that the first 1.13 Skript update was a month ago, and the last Skungee update was also a month ago ¯\_(ツ)_/¯

**Comment by TheLimeGlass** [comment]
Skungee handles the majority of it's processing on the Bungeecord, not Spigot. If the syntax are not working, there will most likely be a console error, that is what I need. Skungee hasn't been tested on the alpha versions either, but doesn't use Skript that much, just to relay the information.

**Comment by TheLimeGlass** [comment]
Fixed in 1.0.7e
